style.css

```diff
@@ -1,37 +1,7 @@
 body {
     background-color: black;
-}
-
-#title-box-wrapper {
-    width: 431px;
-    height: 28px;
-    background-color: lightgrey;
-    margin:auto;
-    /* margin-left: 370px;  */
-}
-
-#title-box {
-    text-align: left;  
-    font-size: 16px;
-    font-family: 'Helvetica Neue';
-}
-
-#buttons {
-    width: 431px;
-    height: 18 px;
-    display: block;
-    margin: auto;
-    background-color: white;
-}
-
-#start-button {
-    /* margin-left: 370px; */
-    display: inline-block;
-}
-
-#stop-button {
-    /* margin-left: 10px; */
-    display: inline-block;
+    display: flex;
+    justify-content: center;
```

Comment:
we usually want to style the body as little as possible, it's better to create a div that takes up 100% of the body and style that. This makes it easier to move the entire page into new environments that already have bodies, or don't let you style them

Reply:
That makes sense, I'll see if I can put it in a new div